session/handler.go

```diff
@@ -153,6 +161,48 @@ func (h *Handler) whoami(w http.ResponseWriter, r *http.Request, ps httprouter.P
 	h.r.Writer().Write(w, r, s)
 }
 
+// swagger:parameters adminLogoutIdentity
+// nolint:deadcode,unused
+type adminLogoutIdentity struct {
+	// ID is the identity's ID.
+	//
+	// required: true
+	// in: path
+	ID string `json:"id"`
+}
+
+// swagger:route DELETE /sessions/identity/{id} v0alpha1 adminLogoutIdentity
```

Comment:
Right. That was my initial idea but doing so would introduce import cycle:
```
	imports github.com/ory/kratos/cmd
	imports github.com/ory/kratos/cmd/courier
	imports github.com/ory/kratos/driver
	imports github.com/ory/kratos/continuity
	imports github.com/ory/kratos/identity
	imports github.com/ory/kratos/session
	imports github.com/ory/kratos/identity: import cycle not allowed
```
since we need an access to `session.PersistenceProvider`.  Unless we keep it in `session/handler.go` but change the route path only.

Reply:
Keeping it in the session handler but having an `/identity` path sounds good to me :+1:

Reply:
What @zepatrik said :) Routes do not necessarily need to match packages :)

Reply:
OK. Will fix that then. Its functionality fits `/identity` endpoint better.

Reply:
Done in 713447ee